### DISCUSSION TITLE
Vision plugin is broken in V3

### DISCUSSION THREAD

**Discussion by xndyz** [sanity-io/sanity#3651] [Sanity Studio Support] [answered]
Getting this error which makes it impossible to use Vision with V3:

`Error: Existing client instance cannot be reconfigured - use `withConfig(newConfig)` to return a new client`

Here's the package versions:
```
    "@sanity/vision": "3.0.0-dev-preview.17",
    "sanity": "dev-preview",
```

**Comment by intensr** [comment]
getting the same error
<img width="1784" alt="image" src="https://user-images.githubusercontent.com/6901764/190886318-f2fd9f44-53d1-4f87-bfe3-a26fa083f505.png">
@rexxars

**Comment by ilyagru** [comment]
Hi! Any suggestions on how to resolve this?

**Comment by xanderjl** [comment]
Same error with same package versions as @fandy 😔

**Comment by kmelve** [comment]
Hi all! Thanks for reporting! 

I'm now seeing that we indeed have another bug here. Let me follow up with the team. 

~This happens because we shipped a breaking change in [`dev-preview.16`](https://github.com/sanity-io/sanity/releases/tag/v3.0.0-dev-preview.16) where the `useClient` hook now requires you to specify the API version (for example `useClient({apiVersion: '2022-09-14'})`). We have updated most plugins to be compatible, so make sure you update.~

**Comment by rexxars** [comment, marked as answer]
Fixed in [v3.0.0-dev-preview.20](https://github.com/sanity-io/sanity/releases/tag/v3.0.0-dev-preview.20) - sorry about that!
